.github/ISSUE_TEMPLATE.md

```diff
@@ -27,4 +27,4 @@ You do NOT have to include this information if this is a FEATURE REQUEST
 
 **- What is the expected behavior?**
 
-**- Please mention your node.js, and operating system version.**
+**- Please mention your CMS, node.js, and operating system version.**
```

Comment:
Does anyone care if this is a separate question or combined?

Reply:
Combined seems fine to me.